client/python/kserve-api/samples/http_infer_binary_resnet.py

```diff
@@ -0,0 +1,139 @@
+#

```

Comment:
perhaps we should have a version showing how to send binary data with jpeg content and a separate sample for other types using the REST extension?